Cargo.toml

```diff
@@ -58,7 +58,7 @@ resolver = "2"
 [workspace.package]
 rust-version = "1.78.0"
 edition = "2021"
-version = "14.3.1"
+version = "14.4.0"
```

Comment:
I suspect this should be 15.0 given [there's breaking changes in the crashtracking API](https://github.com/DataDog/dd-trace-rb/pull/4255).

Reply:
Makes sense.
We also increased the minimum rust version to 1.78 which drops support for windows 7 (which none of our products currently support anyway)

Reply:
I fixed the PR to bump to 15.0.0